### PR TITLE
Feature/dog form

### LINKED ISSUE #9: FE: Setup/Navbar

### LINKED ISSUE #10: FE: Allow user to submit the Register Pet form to add a pet to the user's profile

### PR DESCRIPTION
**Pull request by aominhlong**
## Description 
This PR adds rendering of all dogs on a user profile, adds a refetch to update the user, and add fetch network policy for graphQL. 

#### Where to start?
Start in the 'App' file to see all forms of state and the 'loginUser' function in the 'App' file to see the refetch occur. 

#### How to test?
In the project directory, you can run:

### `npm start`
Runs the app in the development mode.\
Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
The page will reload when you make changes.
- Login with a user and see any existing dogs. 
- Create a dog using the dog forum and see the dog populate under the 'PETS' where the user info exists

#### Issues
Closes #9 
Closes #10 

Optional: 
#### Context
Added a conditional where it will render if 'user.dogs' exists

#### Screenshots (if appropriate)
image

#### Type of Change:
- [] Bug fix (non-breaking change which fixes an issue)
- [] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
What steps have you taken to ensure the code is effective?
- [x] Have you used Dev Tools/ ran code in the console?
- [] Have you ran the Cypress testing suite?

#### Checklist:
- [] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- [x] I have added reviewers.

### CODE REVIEW

**Review by JoshMallery — APPROVED** [review]
Great bug fix on the rendering issue on the user.dog, glad you all got it figured out.

File: src/Info.js [hunk 1]
```diff
@@ -2,21 +2,29 @@ import React from 'react'
 import './Info.css'
 
 const Info = ({ user }) => {
-    return (
-        <div className='user-info-container'>
-            <div className='user-info'>
-                <h2>User Information</h2>
-                <p>Username: {user.username}</p>
-                <p>Email: {user.email}</p>
+    if (user.dogs) {
```
Comment (DanielN88) [inline src/Info.js:5]: This was very simple fix that seems to be working well with our invalid map error.

File: src/App.js [hunk 2]
```diff
@@ -47,15 +97,52 @@ const loginUser = (username, email) => {
...
+  if (errorUser) {
+    console.log(errorUser)
+  }
+}
 
 
-  if (error) {
-    console.log(error)
+const registerDog = (name,age,breed,skills) => {
+  console.log(name,age,breed,skills)
```
Comment (DanielN88) [inline src/App.js:116]: Let's remember to pull out all of these console logs later.

**Review by DanielN88 — APPROVED** [review]
Everything is functioning as intended. Merge when ready.

**Pull request merged**